tests/src/org.ehcache/ehcache/3.10.8-jakarta/src/test/java/org_ehcache/ehcache/EhcacheTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_ehcache.ehcache;
+
+import org.ehcache.Cache;
+import org.ehcache.CacheManager;
+import org.ehcache.PersistentCacheManager;
+import org.ehcache.config.builders.CacheConfigurationBuilder;
+import org.ehcache.config.builders.CacheManagerBuilder;
+import org.ehcache.config.builders.ExpiryPolicyBuilder;
+import org.ehcache.config.builders.ResourcePoolsBuilder;
+import org.ehcache.config.units.EntryUnit;
+import org.ehcache.config.units.MemoryUnit;
+import org.ehcache.xml.XmlConfiguration;
+import org.junit.jupiter.api.Test;
+
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+class EhcacheTest {
```

Comment:
- I don't want to pay attention to `org.ehcache:ehcache-transactions`, OffHeap and `-XX:MaxDirectMemorySize`, GraalVM reachability metadata related to Clustered Cache in this PR, because those three functions seem to be suitable for separate discussion, and Unit testing is complicated.